src/messaging/rabbitmq/consumer.ts

```diff
@@ -90,15 +96,58 @@ export default class RMQConsumer {
     })
   }
 
-  public async cleanUp(): Promise<void> {
-    this.isConnClosing = true
-    if (this.channel != null) {
-      await this.channel.close()
-      console.log(`[Consumer ${this.name}]: Closed channel successfully`)
-    }
-    if (this.conn !== null) {
-      await this.conn.close()
-      console.log(`[Consumer ${this.name}]: Closed connection successfully`)
+  public async cleanUp(isConnClosing = false): Promise<void> {
+    this.isConnClosing = isConnClosing
```

Comment:
[handleConnectionClose](https://github.com/shardeum/collector/pull/55/files?diff=unified&w=1#diff-32657efca8a5803d1a961113054021a61811d80a9c357f371c9b770bb9670c46R161) logic doesn't trigger reconnection if exit event was received. 
Earlier cleanup was only invoked in case of exit events but with recent change, we plugged-in cleanup in reconnection flow also. So, based on where it is invoked, it will be true or false(default).